BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -247,6 +247,24 @@
   gPlatformCommonLibTokenSpaceGuid.PcdUsbKeyboardPollingTimeout | $(USB_KB_POLLING_TIMEOUT)
   gPlatformCommonLibTokenSpaceGuid.PcdLowestSupportedFwVer		| $(LOWEST_SUPPORTED_FW_VER)
 
+  !if $(RSA_SIGN_TYPE) == RSA2048
+    gPlatformCommonLibTokenSpaceGuid.PcdRsaSignType | 0x1
+  !elseif $(RSA_SIGN_TYPE) == RSA3072
+    gPlatformCommonLibTokenSpaceGuid.PcdRsaSignType | 0x2
+  !endif
```

Comment:
BuildLoader.py sets RSA_SIGN_TYPE. But I think setting PcdRsaSignType directly in BuildLoader.py would be better and each board would override the value. We don't have to use RSA_SIGN_TYPE and these conditions here.

Reply:
RSA_SIGN_TYPE and others set in BuildLoader.py, they can be override by each board config values. Default values are set in BuildLoader.py.
I would check the option for setting PcdRsaSignType directly in BuildLoader.py directly